### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFBrush_PredefinedBrushes_ScreenReader: Using NVDA the entire content available in the main window is not announced to the screen reader users.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1453]
Copied from VSO [947720:](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/947720)

Environment Details:
Application Name: .NET Core 3.0 (WPF and ASP.NET Core)

Application version: VS 2019 Preview 4.7.03190

Windows Version: Windows10

ScreenReader Version: Narrator, NVDA 2019.1.1

**Pre-requisite:**

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link. 

**Repro Steps:**

1) Open Visual Studio.

2) Hit on File and select Open->Project/Solution.​
3) Select the "Brushes" solution and open it. ​
4) From the Solution Explorer, right click on the file under the solution "Brushes".​
5) Select "Debug" and hit "Start new Instance".​
6) Main Window for "Brush Examples" solution will appear.​
7) Navigate to "Predefined Brushes" option from left navigation and hit enter key.
8) Main window for "Predefined Brushes" option will appear.
9) Start screen reader.

10) Navigate to the main window and listen if any of the content is announced or not.

Actual:

Using NVDA when screen reader users navigate through the main window, the text content available is not announced to the screen reader users.

Same issue is occurred in below scenario:

1) Solid Color Brush - Syntax

2) Introduction

3) Brush Property - Opacity

4) Brush Property - Transform

5) Gradient Brushes

6) Gradient Spread

7) Pen Examples - Dashes

8) Linear Gradient - Animation

9) Radial Gradient Brush

10) Radial Gradient Brush Animation

Expected:

All the content available in the main content area should be announced to the screen reader users while navigation.

User Impact:

If all the content available in the main content area is not announced to the screen reader users then the information that is available for the visual users will not be available for the screen reader users.

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Comment by rladuca** [comment]
Narrator reads the content just fine if you ask it to (via Narrator + R).  The full contents of this window are not text, they are individual controls, so you have to tell narrator to read these out explicitly.  This isn't a bug in WPF or even in Narrator.
